perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/RegistrarManager.java

```diff
@@ -229,6 +229,19 @@
 	 */
 	List<ApplicationFormItemData> createApplication(PerunSession user, Application application, List<ApplicationFormItemData> data) throws PerunException;
 
+	/**
+	 * Creates a new application and if succeeds, trigger validation and approval.
+	 *
+	 * <p>The method triggers approval for VOs with auto-approved applications.
+	 *
+	 * @param session user present in session
+	 * @param application application
+	 * @param data data
+	 * @return submitted application
+	 * @throws PerunException
+	 */
+	Application submitApplication(PerunSession session, Application application, List<ApplicationFormItemData> data) throws PerunException;
```

Comment:
We probably can mark old ``createApplication()`` as deprecated (in core and RPC javadoc). I don't think anybody else is using it directly.

Reply:
Done